### ISSUE TITLE
Grey font color after PDF export

### ISSUE THREAD

**Issue by Max-194** [atom-community/markdown-preview-plus#512]
After writing down some markdown lines and exporting the preview to PDF, the font color appears to be light grey and not black. Is there a way to change this (and maybe the font size)? Do you configure it in the stylesheet?
I'm kind of new to atom and markdown and can't really find a solution to this.

**Comment by lierdakil** [comment]
You'll have to explain your setup in more detail, because on Atom v1.52 by default this doesn't happen. If anything, pdf font seems to be darker (likely because it looks bolder due to how pdfjs draws it)
![image](https://user-images.githubusercontent.com/7275622/98019868-54bfcd00-1e13-11eb-9397-a4bfe8d1d109.png)

**Comment by Max-194** [comment]
My Atom version is v1.52.0 and on default.
It only happened to me when using the preview in Atom's dark mode, i.e. One dark.
When previewing the markdown file and exporting it to pdf via right click on the preview window, the font color in the final pdf somehow adapted the font color from the dark mode UI, which was kind of greyish in my case.
In light mode this wouldn't happen and the final pdf font color is usually black.

I got it fixed more or less with this addition to Atom's stylesheet:
```
body:not([class]) {
  font-size: 14px;
  color: #000;
  background-color: whitesmoke;
  font-family: Helvetica;
}
h1,h2,h3,h4,h5,h6 {
  color: #000;
}
```
The downside is when opening the preview in dark mode, the preview window is now "white smoke".
But that's the closest I could get to having a exported pdf with black font color and using (half of) Atom in dark mode.
<img width="1552" alt="Bildschirmfoto 2020-11-07 um 15 47 25" src="https://user-images.githubusercontent.com/73878030/98444165-8cc75880-2110-11eb-9e08-e21f13efa77b.png">

**Comment by lierdakil** [comment]
Okay. I still can't reproduce. If `use GithHub.com style` is enabled, the preview will be black-on-white, unless you've added/enabled style overrides. If it's disabled, pdf export should include the background, i.e. it's expected to see something like this:
![image](https://user-images.githubusercontent.com/7275622/98469020-447e6800-21ee-11eb-9ffc-1096027a1bd4.png)

But you might be interested in [this](https://github.com/atom-community/markdown-preview-plus/blob/master/docs/styling-preview.md). TL;DR you can target specific preview modes (i.e. pdf export) in your stylesheet if you're so inclined.

Side note: some style-related inconsistencies when using pandoc backend fixed in v4.8.4.

**Comment by lierdakil** [comment]
I've released v4.10.0 with the ability to use different stylesheet for PDF export and the preview itself and more sensible defaults for PDF export. TL;DR, pdf export now uses 'GitHub light' stylesheet by default regardless of Atom UI theme. There still might be some bugbears hiding in the shadows. Do let me know if you find any.

Also, feel free to post your `default.less` version if you think it can be preferable to 'github.com light' stylesheet. I'd be happy to add it as an alternative preview stylesheet.
